packages/chat/src/utils/messageUtil.ts

```diff
@@ -45,3 +45,27 @@ const twitchMessageTypes = new Map<string, MessageConstructor>([
 export function parseTwitchMessage(rawLine: string): Message {
 	return parseMessage(rawLine, undefined, twitchMessageTypes, true);
 }
+
+export function autoSplitterWithSpaces(text: string, maxMsgLength: number): string[] {
```

Comment:
Function names should be verbs according to the [conventions](https://github.com/twurple/twurple/blob/main/CONTRIBUTING.md#general-code-conventions). How about `splitOnSpaces`?